client/src/components/Link.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Link = ({ className, href, children }) => {
```

Comment:
where is this component used?

Reply:
Sorry, I just deleted it. It was one of my tries, and I forgot to delete it.